lib/rollbax/item.ex

```diff
@@ -4,10 +4,15 @@ defmodule Rollbax.Item do
     %{"access_token" => token,
       "data" => %{
         "server" => %{
-          "host" => List.to_string(host)},
+          "host" => List.to_string(host)
+        },
         "environment" => envt,
-        "language"  => "Elixir",
-        "framework" => "OTP"}}
+        "language"  => language(),
+        "framework" => framework(),
```

Comment:
@whatyouhide it doesn't require much expertise here. :smile:
Rollbar has few special keys in the payload: `language`, `platform`, `framework`.
Apparently we should put Elixir version in the `language` field and Erlang/OTP – `platform`.
The main question is what shall we do with OS info?
* introduce new field in payload, as I see it's will be parsed and displayed with no problem
* somehow append it to the Erlang/OTP (currently in PR)
* drop it for a while

Not sure about appending as it looks clunky:
```
"Erlang/OTP 18 [erts-7.1] [source] [64-bit] [smp:4:4] [async-threads:10] [hipe] [kernel-poll:false] [dtrace] on Unix/Darwin 14.5.0"
```
Honestly I'm more in favor of dropping it. :neutral_face: